.teamcity/_self/projects/WPComTests.kt

```diff
@@ -0,0 +1,130 @@
+package _self.projects
+
+import _self.bashNodeScript
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.perfmon
+import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.BuildFailureOnMetric
+import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.failOnMetricChange
+
+object WPComTests : Project({
+	id("WPComTests")
+	name = "WPCom Tests"
+	description = "Builds which test WordPress.com functionality, such as the Gutenberg plugin."
+
+	params {
+		param("docker_image", "%docker_image_e2e%")
+		param("build.prefix", "1")
+	}
+
+	buildType(Gutenberg)
+})
+
+
+private object Gutenberg : BuildType({
+	name = "Gutenberg tests (desktop)"
```

Comment:
Probably needs the ID so it doesn't loses the history. According to the UI, it is `calypso_Gutenberg`